src/integrationTest/java/uk/gov/hmcts/reform/sendletter/tasks/SerialTaskRunnerTest.java

```diff
@@ -0,0 +1,65 @@
+package uk.gov.hmcts.reform.sendletter.tasks;
+
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.test.autoconfigure.jdbc.AutoConfigureTestDatabase;
+import org.springframework.boot.test.autoconfigure.orm.jpa.DataJpaTest;
+import org.springframework.test.context.junit4.SpringRunner;
+
+import java.sql.SQLException;
+import javax.sql.DataSource;
+
+import static org.mockito.Mockito.mock;
+import static org.mockito.Mockito.never;
+import static org.mockito.Mockito.times;
+import static org.mockito.Mockito.verify;
+
+@RunWith(SpringRunner.class)
+@AutoConfigureTestDatabase(replace = AutoConfigureTestDatabase.Replace.NONE)
+@DataJpaTest
+public class SerialTaskRunnerTest {
+    @Autowired DataSource source;
+
+    @Test
+    public void runs_task_if_not_running() throws SQLException {
```

Comment:
?

Reply:
Perhaps a better name would be 'runs_task'?

Reply:
Or `runs_task_if_that_task_is_not_running_yet` or sth?